readthedocs/settings/dev.py

```diff
@@ -47,6 +47,9 @@ def DATABASES(self):  # noqa
         'test:8000',
     )
 
+    DOCKER_ENABLE = True
```

Comment:
I think this shouldn't be here, at least for now, since we don't have a good guide for development using docker, new contributors wouldn't have this easy.

Reply:
Oh! Sorry. I did not want to commit the settings file